README.md

```diff
@@ -1027,6 +1042,66 @@ service:
     - echo "here I can run custom commands, but no preparation needed for this repo"
 ```
 
+## Custom Linters
+Some people and organizations may choose to have custom made linters run as a part of golangci-lint. That functionality 
+is supported through go's plugin library.
+
+### Create a Copy of `golangci-lint` that Can Run with Plugins
+In order to use plugins, you'll need a golangci-lint executable that can run them. The normal version of this project 
+is built with the vendors option, which breaks plugins that have overlapping dependencies.
+
+1. Download the `i473` branch of https://github.com/dbraley/golangci-lint/tree/i473
+2. From that projects root directory, run `make vendor_free_build`
+3. Copy the `golangci-lint` executable that was created to your path, project, or other location
+
+### Configure Your Project for Linting
+If you already have a linter plugin available, you can follow these steps to define it's usage in a projects 
+`.golangci.yml` file. An example linter can be found at https://github.com/dbraley/example-linter. If you're looking for 
```

Comment:
please point to `github.com/golangci/example-plugin-linter`. I will fork your repo into it.